src/utils/messenger.h

```diff
@@ -39,6 +42,12 @@ class Messenger {
             return;
         }
 
+        bf16_enable = (getenv("XFT_ONECCL_BF16") ? atoi(getenv("XFT_ONECCL_BF16")) : 0);
+        if (bf16_enable) {
+            printf("got 'XFT_ONECCL_BF16=%d', enable BF16 dtype comm.\n", bf16_enable);
+            buf_bf16 = new bfloat16_t[MAX_BF16_BUFFER];
```

Comment:
suggest using aligned_alloc or malloc.
as new will initialize all the data to 0, which is not needed here.

Reply:
Please add this macro XFT_ONECCL_BF16 into wiki.